packages/lib/src/vis/line/LineVis.tsx

```diff
@@ -79,7 +80,11 @@ function LineVis(props: Props) {
   auxArrays.forEach((arr) => assertDataLength(arr, dataArray, 'auxiliary'));
 
   const abscissas = useMemo(() => {
-    return abscissaValue ?? range(dataArray.size);
+    if (abscissaValue) {
+      return isTypedArray(abscissaValue) ? [...abscissaValue] : abscissaValue;
```

Comment:
The conversion of axis values to `number[]` in `getAxisValues()` that I pointed out further up only applies to `HeatmapVis`. I do the same here.

Reply:
I would have reverted the condition to early return `range(dataArray.size)` but no big deal

Reply:
Sounds good :+1: